src/contact.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/no-cycle */
```

Comment:
Instead of disabling this linter rule, let's avoid unnecessary [dependency cycles](https://en.wikipedia.org/wiki/Circular_dependency)

Create a different module for event listeners where you can import all the views modules for `home`, `contact`, etc